Add Bots

hackerkid opened the pull request:
<!-- Please fill in the **bold** fields and tick all applicable boxes by placing an "x" inside "[ ]". -->

**https://github.com/hackerkid/bots**

**The list contains useful resources (frameworks, libraries, bot stores, tutorials etc) on building bots for various platforms.**

- [X] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [X] This pull request has a descriptive title.
- The list I added
	- [X] has been around for at least 14 days,
	- [X] is not a duplicate,
	- [X] only has awesome items with descriptions,
	- [X] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [X] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license),
	- [X] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [X] has been checked for inconsistent formatting, spelling and grammar, and
	- [X] has been added at the bottom of the appropriate category.

hackerkid commented:
@sindresorhus Looks like you updated the guidelines. Should I close the PR or wait for the list to reach 20. The list is 14 days old.

sindresorhus commented:
Some feedback:

- The `Libraries` link in the table of contents has a typo.
- `Selected Tutorials` => `Tutorials`. Everything in the list are selected.
- End all descriptions in a `.`
- `node.js` => `Node.js`

-

> Should I close the PR or wait for the list to reach 20. The list is 14 days old.

No, it's fine. It only applies to new PRs.

sindresorhus commented on readme.md:
```diff
@@ -426,6 +426,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)
 - [Colorful](https://github.com/Siddharth11/Colorful) - Resources to choose your next color scheme.
 - [Steam](https://github.com/scholtzm/awesome-steam) - Digital distribution platform.
+- [Bots](https://github.com/hackerkid/bots) - Resources on building bots.
```
Don't like everything to be prefixed with "resources". All lists contain resources. Maybe just `Building bots` or something.

hackerkid commented:
@sindresorhus 

> - The Libraries link in the table of contents has a typo.
> - Selected Tutorials => Tutorials. Everything in the list are selected.
> - End all descriptions in a .
> - node.js => Node.js

Fixed in https://github.com/hackerkid/bots/commit/05fe55f46b870af7e7625188005bf4ca9a655e5b

> Don't like everything to be prefixed with "resources". All lists contain resources. Maybe just Building bots or something.

Fixed in https://github.com/sindresorhus/awesome/pull/692/commits/8a7fd37530dc734d0b8be0f87570ca1b0aeb49e5

Thanks for the feedback. :smile:

sindresorhus commented:
Very good. Landed :)
